src/pages/FilesPage/FilesTabs/DatasetsPane/DatasetsPane.tsx

```diff
@@ -70,6 +70,13 @@ export const DatasetsPane: FC = function DatasetsPane() {
           datasets={datasets}
           isLoading={isLoading}
           onSelectDataset={setSelectedDataset}
+          onUpdate={(name, datasetId) => {
+            const newDataset = datasets?.find(dataset => dataset.id === datasetId);
+
+            if (newDataset !== undefined) newDataset.title = name;
```

Comment:
it is dirty trick.
You will change this code immediately after more editable fields will be introduced.